app/scripts/controllers/metametrics.js

```diff
@@ -530,10 +534,12 @@ export default class MetaMetricsController {
         );
       }
 
-      const combinedProperties = merge(
-        payload.sensitiveProperties,
-        payload.properties,
-      );
+      const combinedProperties = merge(payload.sensitiveProperties, {
+        ...payload.properties,
+        ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+        accountAddress: this.selectedAddress,
```

Comment:
Is it critical that this be tracked in MMI? This is personally identifiable information. Generally we don't track that.

This conflicts with our plans in the near term to get rid of the concept of a "globally selected address" as well.

Reply:
@shane-t can you please advise on how critical it is for us having this address? It has been added [in here](https://github.com/ConsenSys/mmi-extension/pull/340). Maybe we can remove it for now and try to figure out if we can get it from other place?

Reply:
Let's remove this @zone-live 
While we have a separate user agreement in MMI, we're not dependent on this for our metrics, and it would introduce ambiguity if someone is reading the codebase. Moreover, if the concept of a global selected address is being deprecated, it will only cause issues in the future.

Reply:
`accountAddress` removed from the code